compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -312,7 +312,13 @@ object Splicer {
 
     protected final def interpretTree(tree: Tree)(implicit env: Env): Result = tree match {
       case Apply(TypeApply(fn, _), quoted :: Nil) if fn.symbol == defn.QuotedExpr_apply =>
-        interpretQuote(quoted)
+        val quoted1 = quoted match {
+          case quoted: Ident if quoted.symbol.is(InlineProxy) && quoted.symbol.is(MethodFlag) => // inline proxy for by-name parameter
```

Comment:
Move comment to next line to shorten

Reply:
You can just use `quoted.symbol.is(InlineMethod)`. Then you don't need to rename `Method` to `MethodFlag`.

Reply:
Actually this is is not an `InlineMethod` which is `Inline | Method`. This one is `InlineProxy | Method` which could mark the `def` binding for by-name parameters. I will create `InlineByNameProxy` to make the intent clear.